src/sql/src/statements/describe.rs

```diff
@@ -32,6 +34,13 @@ impl DescribeTable {
     }
 }
 
+impl Display for DescribeTable {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let name = self.name();
+        write!(f, r#"DESCRIBE TABLE {name}"#)
```

Comment:
I think it's better to always quote the database names, table names and column names with `"`.

Reply:
The underneath Ident impls:

```rust
impl fmt::Display for Ident {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        match self.quote_style {
            Some(q) if q == '"' || q == '\'' || q == '`' => {
                let escaped = value::escape_quoted_string(&self.value, q);
                write!(f, "{q}{escaped}{q}")
            }
            Some('[') => write!(f, "[{}]", self.value),
            None => f.write_str(&self.value),
            _ => panic!("unexpected quote style"),
        }
    }
}
```